zerver/webhooks/stripe/view.py

```diff
@@ -53,6 +55,14 @@ def api_stripe_webhook(
     payload: Dict[str, Any] = REQ(argument_type="body"),
     stream: str = REQ(default="test"),
 ) -> HttpResponse:
+    sig_header = request.META["HTTP_STRIPE_SIGNATURE"]
```

Comment:
Before we do any of this, we should check whether the `stripe_webhook_endpoint_secret` is even set, if the user didn't set it, then the `construct_event` call would fail.

Reply:
@eeshangarg if there is no `stripe_webhook_endpoint_secret` should I return a `HTTP 400` or just continue sending webhook without signature verification ?